layouts/list.algolia.json

```diff
@@ -1,9 +1,9 @@
 {{- $.Scratch.Add "index" slice -}}
 {{- $section := $.Site.GetPage "section" .Section }}
 {{- range .Site.AllPages -}}
-  {{- if or (and (.IsDescendant $section) (and (not .Draft) (not .Params.private))) $section.IsHome -}}
+  {{- if or (and (.IsDescendant $section) (and not .Params.private)) $section.IsHome -}}
```

Comment:
Could the removal of the parenthesis be the reason the build is failing? I'm just trying to figure out every possible cause. Then again, I'm not sure about this general syntax that the algolia command needs.

Reply:
I can try to push a change that brings back the private param 🤔